api/provider/aws/service_create_test.go

```diff
@@ -0,0 +1,64 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/quintilesims/layer0/api/tag"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestService_createTags(t *testing.T) {
+	tagStore := tag.NewMemoryStore()
+	service := NewServiceProvider(nil, tagStore, nil)
+
+	if err := service.createTags("svc_id", "svc_name", "env_id"); err != nil {
+		t.Fatal(err)
+	}
+
+	expected := models.Tags{
```

Comment:
ditto - need to add `load_balancer_id` tag

Reply:
✅